---
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: true
    number_sections: true
    fig_caption: true
    toc: false
    includes:
        before_body: 
            header.sty
geometry: "left=2.5cm,right=2.5cm,top=2cm,bottom=2cm"
linestretch: 1.2
fontsize: 12pt
indent: true
bibliography: refs.bib
citation-style: ../csl/nature-publishing-group-vancouver.csl
link-citations: true
citecolor: teal
linkcolor: teal
urlcolor: teal
header-includes:
- \usepackage[labelsep=period]{caption}
- \usepackage[labelfont=bf]{caption}
- \usepackage{booktabs}
- \usepackage{caption}
- \usepackage{microtype}
- \usepackage{sectsty}
- \captionsetup[figure]{font=small}
- \captionsetup[table]{font=small}
- \captionsetup[table]{justification=justified}
- \captionsetup[figure]{justification=justified}
- \usepackage[default]{sourcesanspro}

editor_options: 
    markdown:
      canonical: true 
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, fig.align = "center")
```

```{r load-pkgs, include=FALSE}
library(tidyverse) # Load the tidyverse. Functions are used for tidying data and creating plots via ggplot2.

library(kableExtra) # Load kableExtra. Table generation.

library(wordcountaddin) # Load wordcountaddin. Counts words and characters in text in an R markdown document.

library(rmdwc)

citation_count <-
  rmdcount(
    files = "report.Rmd",
    space = "[:space:]",
    word = "\\@",
    line = "\\n",
    exclude = "```\\{.*?```"
  )
```

```{r child = "abstract.Rmd"}
```

```{=tex}
\normalsize
\begin{flushright}
`r word_count("abstract.Rmd")` Words
\end{flushright}
\hrulefill\\
\setlength{\parindent}{10pt}
```
# Introduction

## FOXA1 Expression and ERα+ Breast Cancer

## tRNAs and Gene Expression

------------------------------------------------------------------------

# Materials & Methods

## Acquisition of Public ChIP-seq Datasets

ChIP-seq was performed on genetically modified MCF7L cells (*insertion, using a lentiviral cDNA delivery system to express Dox-inducible FOXA1*) [@fu2019].
Datasets were deposited into the National Centre for Biotechnology Information (NCBI) Sequence Read Archive (SRA)[@leinonen2010] under accession no.
PRJNA512997 (Table \@ref(tab:data)).
Using "Genetic Manipulation Tools" within the Galaxy[@thegala2022] environment (v 23.0.rc1), SRAs were converted to FastQ files.
FastQ files were then aligned to the human genome assembly GRCh37 (hg19) using Bowtie2 (v 2.5.0)[@langmead2012] to output BAM files.

```{r data}
kbl(
  data.frame(
    "Experiment" = c(
      "PRJNA512997",
      "",
      "",
      "",
      "",
      "",
      ""
    ),
    "SRA" = c(
      "SRR8393424",
      "SRR8393425",
      "SRR8393426",
      "SRR8393427",
      "SRR8393428",
      "SRR8393431",
      "SRR8393432"
    ),
    "Factor" = c(
      "FOXA1",
      "",
      "",
      "H3K27ac",
      "",
      "None (input)",
      ""
    ),
    "Tissue" = c(
      "MCF-7LP",
      "",
      "",
      "",
      "",
      "",
      ""
    ),
    "Assembly" = c(
      "GRCh37 (Hg19)",
      "",
      "",
      "",
      "",
      "",
      ""
    )
  ),
  booktabs = TRUE,
  caption = "Publicly available ChIP-seq SRA files aquired from the NCBI SRA database (accession no. PRJNA512997).",
  format = "markdown"
)
```

### EaSeq for Chip-seq Peak Quantification

BAM files were uploaded into EaSeq (v1.111) as "Datasets" using the standard settings for Chip-seq data.
GRCh37 (hg19) tRNA sequences (n = 606) were downloaded as a "Geneset" from the UCSC Table Browser[@Karolchik2004], (available at <https://genome.ucsc.edu>).
High-confidence tRNAs (n = 416) identified in the GtRNAdb[@Chan2016] were extracted as a "Regionset".

Signal peak intensities surrounding tRNAs were quantified using the EaSeq "quantify" tool.
Here the default settings "Normalize to reads per million" and "Normalize counts to DNA-fragments" were left checked.
The default setting "Normalise to a signal of 1000 bp" was unchecked.
The window size was offset ±500bp from the start of each tRNA gene.
Outputs are referred to as "Q-values".

To quantify upstream and downstream signals, the "quantify" tool was used with adjusted window sizes.
The upstream region was defined as 500 bp preceding and the first nucleotide of tRNA loci.
Thus, the start position was offset to 0 bp, and the end position was offset to -500 bp.
The downstream region constitutes the 500 bp region beginning with the first nucleotide of tRNA gene body.
The start position was offset to 1 bp, and the end position was offset to 500 bp.

Following quantification, data was visualised with "heatmap", "average", and "overlay" EaSeq tools.
EaSeq[@lerdrup2016] is avaiable at <http://easeq.net>.

## Motif Analysis

## Statistics

Statistical tests and graphs were generated with R[@r] (v 4.2.3), R Studio[@rstudio] (v 2023.03.0.386) and the tidyverse[@wickham2019] package.

------------------------------------------------------------------------

# Results

## FOXA1 and H3k27ac Binding Events

### Why?

### What?

-   quantification of reads

<!-- -->

-   heatmaps

-   ordered by -dox q-value

-   average tracks

### Results

-   foxa1 at all tRNAs

-   H3k27ac at all but 2 genes

### Suggests

## Localisation of FOXA1 at indicidual tRNA genes in MCF-7 cells

```{r clusters, out.width='90%'}
knitr::kable(
  data.frame(
    "Group" = c(
      "ALOXE",
      "Ebersole",
      "HES7",
      "Per1",
      "TMEM107",
      "Arg-CCG",
      "Glu-TTC",
      "iMET",
      "Met",
      "SeC"
    ),
    "Function" = c(
      "Insulator Function[@raab2011; @sizer2022]",
      "Insulator Function[@Ebersole2011; @sizer2022]",
      "",
      "",
      "Insulator Function [@raab2011; @sizer2022]",
      "Implicated in Cancer Progression[@Goodarzi2016]",
      "Implicated in Cancer Progression[@Goodarzi2016]",
      "Proliferation of Breast Cancer",
      "iMet Control",
      "Involved in REDOX [@Sangha2022]"
    )
  ),
  booktabs = TRUE,
  caption = ".",
  format = "markdown"
) 
```

------------------------------------------------------------------------

# Discussion

-   not compared mcf-7 to treatment responsive cells

## Future

-   FOXA1 alone not efficient to increase activity

    -   p300

-   FOXA1 moves nucleosomes to make other TF acessible

-   Loses fox = weak binding?

-   Dynamic and stable marks

-   pertubations

<!-- -->

-   ATAC-seq

```{=tex}
\begin{flushright}
`r word_count("report.Rmd") - 2` Words
\end{flushright}
```

------------------------------------------------------------------------

# References
