---
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: true
    number_sections: false
    fig_caption: true
    toc: false
    includes:
        before_body: 
            header.sty
geometry: "left=2.5cm,right=2.5cm,top=2cm,bottom=2cm"
linestretch: 1.2
fontsize: 12pt
indent: true
mainfont: Charter
bibliography: refs.bib
citation-style: ../csl/university-of-york-vancouver.csl
link-citations: true
citecolor: teal
linkcolor: teal
urlcolor: teal
header-includes:
- \usepackage[labelsep=period]{caption}
- \usepackage[labelfont=bf]{caption}
- \usepackage{booktabs}
- \usepackage{caption}
- \usepackage{microtype}
- \usepackage{sectsty}
- \captionsetup[figure]{font=small}
- \captionsetup[table]{font=small}
- \captionsetup[table]{justification=justified}
- \captionsetup[figure]{justification=justified}
- \usepackage{epigrafica}
- \usepackage{epigrafica}
- \usepackage[LGR,OT1]{fontenc}
- \normalfont
editor_options: 
    markdown:
      canonical: true 
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, fig.align = "center", fig.pos = "H", out.width = '100%')
```

```{r load-pkgs, include=FALSE}
library(tidyverse) # Load the tidyverse. Functions are used for tidying data and creating plots via ggplot2.

library(kableExtra) # Load kableExtra. Table generation.

library(wordcountaddin) # Load wordcountaddin. Counts words and characters in text in an R markdown document.

library(rmdwc)

citation_count <-
    rmdcount(
        files = "report.Rmd",
        space = "[:space:]",
        word = "\\@",
        line = "\\n",
        exclude = "```\\{.*?```"
    )
```

```{r child = "abstract.Rmd"}
```

```{=tex}
\normalsize
\begin{flushright}
`r word_count("abstract.Rmd")` Words
\end{flushright}
\hrulefill\\
\setlength{\parindent}{10pt}
```
# Introduction

-   txn by polymerases

-   pol iii

-   pol iii synthesises RNAs, e.g

-   "Recruitment of Pol III to its target genes requires TFIIIB, which is in most cases recruited by TFIIIC, a six-subunit complex that binds two internal promoter motifs, termed the A and B boxes, downstream of the [transcription start site](https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/transcription-initiation-site "Learn more about transcription start site from ScienceDirect's AI-generated Topic Pages") of tRNA genes ([Ramsay and Vannini, 2018](https://www.sciencedirect.com/science/article/pii/S0378111922000993?via%3Dihub#b0170), [Schramm and Hernandez, 2002](https://www.sciencedirect.com/science/article/pii/S0378111922000993?via%3Dihub#b0200))."

# Pol III & cancer

-   aberrant expression = cancers

-   pol ii products linked to cancer e.g.s (tRNAs)

-   tumours express tRNAs that are absent from the normal tissue of origin (Tumour-specific phenylalanine tRNA contains two supernumary methylated bases, Nature)

-   Elevated tRNA~i~^Met^ synthesis can drive cell proliferation and oncogenic transformation, Cell 2008

## ER+ BC

-   75% of cancers

-   treatments - SERM or SERD

-   endocrine resistance

-   ERa nuclear receptor, ERE

-   Thus, ERα is implicated in the control of Pol III transcription in MCF-7 cells.

## FOXA1 Expression and ERα+ Breast Cancer

"\~ 95% of ER- regulated genes require FOXA1 co-occurrence, a pioneer whose experimental ablation in vivo is detrimental to ER binding capacity [9, 10].

"Early studies reported that FoxA1-binding sites overlap with 50% of ER-binding sites (Carroll et al., 2005) and that FoxA1 is required for at least half of all ER-binding events in MCF-7 breast cancer cells (Carroll et al., 2005; Laganie\`re et al., 2005). Later findings via genome-wide analysis (Carroll et al., 2006; Lupien et al., 2008; Hurtado et al., 2011) have been interpreted in support of this general model. In addition, it has been reported that inhibition of ER produces no change in FoxA1 genomic binding patterns (Lupien et al., 2008; Hurtado et al., 2011). These investigations have either focused on a small number of binding locations or compared FoxA1 binding only between unstimu- lated cells and cells treated with an ER antagonist (Hurtado et al., 2011; Lupien et al., 2008). Contrary to these findings, an independent study reported that upon knockdown of ER, FoxA1 binding is lost at many unstimulated ER-binding sites (Caizzi et al., 2014), evidence that ER may in fact regulate binding of FoxA1.."

You are specifically investigating if overexpression of FOXA1 leads to increased tDNA occupancy (more tDNAs bound +/or stronger binding by FOXA1) & activation, as inferred from H3K27ac.

------------------------------------------------------------------------

# Materials & Methods

## Acquisition of Public ChIP-seq Datasets

ChIP-seq was performed on genetically modified MCF7L cells (*insertion, using a lentiviral cDNA delivery system to express Dox-inducible FOXA1*) [@fu2019].

Datasets were deposited into the National Centre for Biotechnology Information (NCBI) Sequence Read Archive (SRA)[@leinonen2010] under accession no.
PRJNA512997 (Table \@ref(tab:data)).
Using "Genetic Manipulation Tools" within the Galaxy[@thegala2022] environment (v 23.0.rc1), SRAs were converted to FastQ files.
FastQ files were then aligned to the human genome assembly GRCh37 (hg19) using Bowtie2 (v 2.5.0)[@langmead2012] to output BAM files.

```{r data}
kbl(
  data.frame(
    "Experiment" = c(
      "PRJNA512997",
      "",
      "",
      "",
      "",
      "",
      ""
    ),
    "SRA" = c(
      "SRR8393424",
      "SRR8393425",
      "SRR8393426",
      "SRR8393427",
      "SRR8393428",
      "SRR8393431",
      "SRR8393432"
    ),
    "Factor" = c(
      "FOXA1",
      "",
      "",
      "H3K27ac",
      "",
      "None (input)",
      ""
    ),
    "Tissue" = c(
      "MCF-7LP",
      "",
      "",
      "",
      "",
      "",
      ""
    ),
    "Assembly" = c(
      "GRCh37 (Hg19)",
      "",
      "",
      "",
      "",
      "",
      ""
    )
  ),
  
  booktabs = TRUE,
  caption = "Publicly available ChIP-seq SRA files aquired from the NCBI SRA database (accession no. PRJNA512997).",
  format = "markdown"
)
```

### EaSeq for Chip-seq Peak Quantification

BAM files were uploaded into EaSeq (v1.111) as "Datasets" using the standard settings for Chip-seq data.
GRCh37 (hg19) tRNA sequences (n = 606) were downloaded as a "Geneset" from the UCSC Table Browser[@Karolchik2004], (available at <https://genome.ucsc.edu>).
High-confidence tRNAs (n = 416) identified in the GtRNAdb[@Chan2016] were extracted as a "Regionset".

Signal peak intensities surrounding tRNAs were quantified using the EaSeq "quantify" tool.
Here the default settings "Normalize to reads per million" and "Normalize counts to DNA-fragments" were left checked.
The default setting "Normalise to a signal of 1000 bp" was unchecked.
The window size was offset ±500bp from the start of each tRNA gene.
Outputs are referred to as "Q-values".

To quantify upstream and downstream signals, the "quantify" tool was used with adjusted window sizes.
The upstream region was defined as 500 bp preceding and the first nucleotide of tRNA loci.
Thus, the start position was offset to 0 bp, and the end position was offset to -500 bp.
The downstream region constitutes the 500 bp region beginning with the first nucleotide of tRNA gene body.
The start position was offset to 1 bp, and the end position was offset to 500 bp.

Following quantification, tRNA binding events were arranged in ascending order -DOX Q-value and visualised as heatmaps.
Data was also visualised with "average", and "overlay" EaSeq tools.

EaSeq[@lerdrup2016] is avaiable at <http://easeq.net>.

## Motif Analysis

Multiple EM for Motif Elicitation ChIP (MEME) Suite

## Statistics

Statistical tests and graphs were generated with R[@r] (v 4.2.3), R Studio[@rstudio] (v 2023.03.0.386) and the tidyverse[@wickham2019] package.

------------------------------------------------------------------------

# Results

## Binding of FOXA1 and H3k27ac to tRNA Genes

To investigate the impact of FOXA1 on tRNA enhancers in ER+ MCF-7 cells, public ChIP-Seq datasets from Fu et al. (2019)[@fu2019] were interrogated.
In this paper, a doxycycline (Dox) inducible OE system was used to achieve FOXA1 OE akin to tamoxifen-resistant (TamR) MCF-7 cells[@fu2019].

FOXA1 and H3K27ac peaks of 416 high-confidence tRNAs were quantified relative to the ±500 bp flanking regions.
Mapped reads of FOXA1 and H3K27ac binding were visualised as heatmaps and ordered by increasing -DOX Q-value.
This revealed a concentration of FOXA1 and H3k27ac at approximately half of tRNAs, relative to ±10 kb flanking regions.
Upon FOXA1 OE, FOXA1 binding increased at a small proportion of tRNAs genes and H3K27ac binding decreases at approximately half of tRNA genes (Figure \@ref(fig:results-1)A).
This was confirmed by average signal intensity plots of FOXA1 and H3K27ac binding (Figure \@ref(fig:results-1)B).
Input reads generated minimal peak enrichment (Supplementary Figure X).

Peaks were classified as binding events if Q-values exceeded input values (Figure \@ref(fig:results-1)C).
FOXA1 interacts with 329 tRNA genes and H3K27ac with 293 tRNA genes.
FOXA1 co-binds with 89.4% of H3K27ac sites.
Upon FOXA1 OE, 40 FOXA1 binding sites are lost and 30 are gained.
H3K27ac sites are lost at 50 tDNAs, and 23 are gained.
Here, FOXA1 co-binding represents 92.5% of H3K27ac sites.

Upon FOXA1 OE, mean Q-values significantly increased 1.18-fold for FOXA1 binding (p \< 0.0001), and significantly decreased 0.86-fold for H3K27ac (p \< 0.01) (Supplementary Figures X).
However, FOXA1 OE leads to a significant difference in FOXA1 and H3K27ac binding between individual tDNAs (p \<0.0001) (Figure \@ref(fig:results-1)D).

Together, these results support the notion that FOXA1 overexpression alters the binding landscape of FOXA1 and H3K27ac at tRNAs.

```{r results-1, out.width = '100%', fig.cap = "(B) Ratiometric heatmaps of the log2 ratio between the binding of FOXA1 or H3k27ac with endogenous FOXA1 expression vs. the binding of FOXA1 or H3k27ac with FOXA1 OE."}
knitr::include_graphics("../images/results-01.png")
```

## Figure 2

### Why?

How does FOXA1 alter H3K27ac binding?

Using Q-values, tRNAs that are differently enriched upon FOXA1 OE were categorised as 'UP' or 'DN' (FOXA1 = .

This discovered substantially more tRNAs with increased (UP) than decreased (DN) FOXA1 (92 vs. 21) (Figure \@ref(fig:results-2)A).
However, for H3K27ac, the number of tRNAs with an increase (UP) was comparable to those with a decrease (DN) (41 vs. 44) (Figure \@ref(fig:results-2)B).

Of the tDNAs which gain (UP) H3k27ac, 51% (21) also gain (UP) FOXA1; none lose (DN) FOXA1.

Of the tDNAs which lose (DN) H3k27ac, 22.7% (10) also lose FOXA1, with just 1 tDNA gaining FOXA1 (Figure \@ref(fig:results-2)C).

Examples of these tRNAs are shown in Figure \@ref(fig:results-2)D.

### 

```{r results-2, out.width='100%', fig.cap = "(B) Ratiometric heatmaps of the log2 ratio between the binding of FOXA1 or H3k27ac with endogenous FOXA1 expression vs. the binding of FOXA1 or H3k27ac with FOXA1 OE."}
knitr::include_graphics("../images/results-02.png")
```

## Figure 3

The next step was to investigate the impact of FOXA1 on the number of active genes.
Almost half of human tDNAs are silent or poorly expressed [@Torres2019].
Thus, tRNAs were classified as 'active' if H3K27ac Q-values exceeded the median -DOX value (Q \> 1.808).

When FOXA1 is over-expressed, the activity status of the majority (87.3%) of genes remain unchanged.
The number of of tDNAs that lose and gain activity are 27 and 26, respectively.

Of the +Dox activated genes, 12 also gained FOXA1.

FOXA1 alone is insufficient in increasing tRNA activity.

Suggests?

```{r results-3, out.width = '100%', fig.cap = "."}
knitr::include_graphics("../images/results-03.png")
```

## Figure ?

-   Relative position (not very interesting)

-   Binding at isotopes?
    Certain AA more up than others?

## Figure 4

#### Why?

### What?

MEME CentriMo to identify de novo motifs that are enriched at tRNAs which gain both FOXA1 and H3k27ac

-   relative to other tRNAS (gain/lose, lose/lose, lose/gain)

-   FOXA1 not enriched

-   Look at ERE, AP-1, others?

-   Top 3 motifs (fisher E values) - IRF7, ERR3, NR4A1

-   A and B box motifs as a control?
    How?

    -   All downstream

    -   tRNAs where 'matching sequences' all best matches = code for valine

    -   Branched amino acids associated with lower BC risk

#### Suggests?

```{r results-4, out.width = '100%', fig.cap = ""}
knitr::include_graphics("../images/results-04.png")
```

## Figure 5

-   Localisation of FOXA1 at individual tRNA genes in MCF-7 cells

Why?

-   tRNAs implicated in cancer

-   Are they upregulated?

-   Look at gain function/gain h3/fox

-   Motif ontology

-   

What?

Suggests?

```{r clusters}
knitr::kable(
  data.frame(
    "Group" = c(
      "ALOXE",
      "Ebersole",
      "HES7",
      "Per1",
      "TMEM107",
      "Arg-CCG",
      "Glu-TTC",
      "iMET",
      "Met",
      "SeC"
    ),
    "Function" = c(
      "Insulator Function[@raab2011; @sizer2022]",
      "Insulator Function[@Ebersole2011; @sizer2022]",
      "",
      "",
      "Insulator Function [@raab2011; @sizer2022]",
      "Implicated in Cancer Progression[@Goodarzi2016]",
      "Implicated in Cancer Progression[@Goodarzi2016]",
      "Proliferation of Breast Cancer",
      "iMet Control",
      "Involved in REDOX [@Sangha2022]"
    )
  ),
  booktabs = TRUE,
  caption = ".",
  format = "markdown"
) 
```

------------------------------------------------------------------------

# Discussion

-   FOXA1 alone not efficient to increase activity

    -   p300

-   FOXA1 moves nucleosomes to make other TF accessible?

-   Brf1 required for iMET txn

-   Loses fox = weak binding?

-   Dynamic vs stable marks

-   ATAC-seq

# Conclusion

```{=tex}
\begin{flushright}
`r word_count("report.Rmd") - 2` Words
\end{flushright}
```

------------------------------------------------------------------------

# References
