---
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: true
    number_sections: true
    fig_caption: true
    toc: false
    includes:
        before_body: 
            header.sty
geometry: "left=2.5cm,right=2.5cm,top=2cm,bottom=2cm"
linestretch: 1.2
fontsize: 11pt
indent: true
bibliography: refs.bib
citation-style: ../csl/nature-publishing-group-vancouver.csl
link-citations: true
citecolor: teal
linkcolor: teal
urlcolor: teal
header-includes:
- \usepackage[labelsep=period]{caption}
- \usepackage[labelfont=bf]{caption}
- \usepackage{booktabs}
- \usepackage{caption}
- \usepackage{microtype}
- \usepackage{sectsty}
- \captionsetup[figure]{font=small}
- \captionsetup[table]{font=small}
- \captionsetup[table]{justification=justified}
- \captionsetup[figure]{justification=justified}
- \usepackage[lf]{venturis}
- \normalfont

editor_options: 
    markdown:
      canonical: true 
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, fig.align = "center")
```

```{r load-pkgs, include=FALSE}
library(tidyverse) # Load the tidyverse. Functions are used for tidying data and creating plots via ggplot2.

library(kableExtra) # Load kableExtra. Table generation.

library(wordcountaddin) # Load wordcountaddin. Counts words and characters in text in an R markdown document.

library(rmdwc)

citation_count <-
  rmdcount(
    files = "report.Rmd",
    space = "[:space:]",
    word = "\\@",
    line = "\\n",
    exclude = "```\\{.*?```"
  )
```

```{r child = "abstract.Rmd"}
```

```{=tex}
\normalsize
\begin{flushright}
`r word_count("abstract.Rmd")` Words
\end{flushright}
\hrulefill\\
\setlength{\parindent}{10pt}
```
# Introduction

## FOXA1 Expression and ERα+ Breast Cancer

## tRNAs and Gene Expression

------------------------------------------------------------------------

# Materials & Methods

## MCF7L ChIP-seq Data from NCBI

ChIP-seq was performed on MCF7L cells expressing Dox-inducible FOXA1, established using a lentiviral cDNA delivery system [@fu2016; @fu2019]. Datasets were deposited into the National Centre for Biotechnology Information (NCBI) Sequence Read Archive (SRA) under accession no. PRJNA512997[@leinonen2010] (Table \@ref(tab:data)). SRAs were downloaded by Galaxy (v23.0.rc1)[@thegala2022] and converted to FastQ files and then into BAM files using 'Genetic Manipulation Tools'.

```{r data}
kbl(
  data.frame(
    "Experiment" = c(
      "PRJNA512997",
      "",
      "",
      "",
      "",
      "",
      ""
    ),
    "SRA" = c(
      "SRR8393424",
      "SRR8393425",
      "SRR8393426",
      "SRR8393427",
      "SRR8393428",
      "SRR8393431",
      "SRR8393432"
    ),
    "Factor" = c(
      "FOXA1",
      "",
      "",
      "H3K27ac",
      "",
      "None (input)",
      ""
    ),
    "Tissue" = c(
      "MCF-7LP",
      "",
      "",
      "",
      "",
      "",
      ""
    ),
    "Assembly" = c(
      "GRCh37 Hg19",
      "",
      "",
      "",
      "",
      "",
      ""
    )
  ),
  booktabs = TRUE,
  caption = "Publicly available ChIP-seq SRA files aquired from the NCBI SRA database (accession no. PRJNA512997).",
  format = "markdown"
)
```

## EaSeq for the Quantification of FOXA1 and H3K27ac at tRNA genes

BAM files were uploaded into EaSeq (v1.111)[@lerdrup2016] as 'Datasets' using the standard settings for Chip-seq data.
The reference genome hg19 was downloaded as a 'Geneset' from the UCSC Table Browser[@Karolchik2004].
High-confidence tRNAs identified in the GtRNAdb[@Chan2016] were extracted as a 'Regionset'.
Signal peak intensities surrounding tRNAs were quantified using the 'Quantify' tool.
Here, the default settings were left checked, with the exception of 'Normalise to a signal of'.
The window size was offset ±500bp from the start of each gene.
Outputs are referred to as 'Q-values'.

-   sorted by q value

-   heatmap, filltrack, ect

## Motif Analysis

## Statistics

------------------------------------------------------------------------

# Results

## Localisation of FOXA1 at tRNA genes in MCF-7 cells

Table \@ref(tab:clusters)

```{r clusters, out.width='90%'}
knitr::kable(
  data.frame(
    "Group" = c(
      "ALOXE",
      "Ebersole",
      "HES7",
      "Per1",
      "TMEM107",
      "Arg-CCG",
      "Glu-TTC",
      "iMET",
      "Met",
      "SeC"
    ),
    "Function" = c(
      "Insulator Function[@raab2011]",
      "",
      "",
      "",
      "",
      "Implicated in Cancer",
      "",
      "Proliferation of Breast Cancer",
      "iMet Control",
      "Involved in REDOX"
    )
  ),
  booktabs = TRUE,
  caption = ".",
  format = "markdown"
) 
```

## Co-localisation of FOXA1 with enhancer marks at tRNA genes

------------------------------------------------------------------------

# Discussion

## Future

-   FOXA1 alone not efficient to increase activity

    -   p300

-   FOXA1 moves nucleosomes to make other TF acessible

-   Loses fox = weak binding?

-   Dynamic and stable marks

-   pertubations

<!-- -->

-   ATAC-seq

```{=tex}
\begin{flushright}
`r word_count("report.Rmd") - 2` Words
\end{flushright}
```

------------------------------------------------------------------------

# References
