---
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: true
    number_sections: true
    fig_caption: true
    toc: false
    includes:
        before_body: 
            header.sty
geometry: "left=2.5cm,right=2.5cm,top=2cm,bottom=2cm"
linestretch: 1.2
fontsize: 11pt
indent: true
bibliography: refs.bib
citation-style: ../csl/nature-publishing-group-vancouver.csl
link-citations: true
citecolor: teal
linkcolor: teal
urlcolor: teal
header-includes:
- \usepackage[labelsep=period]{caption}
- \usepackage[labelfont=bf]{caption}
- \usepackage{booktabs}
- \usepackage{caption}
- \usepackage{microtype}
- \usepackage{sectsty}
- \captionsetup[figure]{font=small}
- \captionsetup[table]{font=small}
- \captionsetup[table]{justification=justified}
- \captionsetup[figure]{justification=justified}
- \usepackage[lf]{venturis}
- \normalfont

editor_options: 
    markdown:
      canonical: true 
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE, fig.align = "center")
```

```{r load-pkgs, include=FALSE}
library(tidyverse) # Load the tidyverse. Functions are used for tidying data and creating plots via ggplot2.

library(kableExtra) # Load kableExtra. Table generation.

library(wordcountaddin) # Load wordcountaddin. Counts words and characters in text in an R markdown document.

library(rmdwc)

citation_count <-
  rmdcount(
    files = "report.Rmd",
    space = "[:space:]",
    word = "\\@",
    line = "\\n",
    exclude = "```\\{.*?```"
  )
```

```{r child = "abstract.Rmd"}
```

```{=tex}
\normalsize
\begin{flushright}
`r word_count("abstract.Rmd")` Words
\end{flushright}
\hrulefill\\
\setlength{\parindent}{10pt}
```
# Introduction

## FOXA1 Expression and ERÎ±+ Breast Cancer

## tRNAs and Gene Expression

------------------------------------------------------------------------

# Materials & Methods

## MCF7L ChIP-seq Data from NCBI

ChIP-seq was performed on MCF7L cells expressing Dox-inducible FOXA1[@fu2019]. Datasets were deposited into the National Centre for Biotechnology Information (NCBI) Sequence Read Archive (SRA) under accession no. PRJNA512997[@leinonen2010] (Table \@ref(tab:data)). SRAs were downloaded by Galaxy (v23.0.rc1)[@thegala2022] and converted to FastQ files and then into BAM files using "Genetic Manipulation Tools".

```{r data}
knitr::kable(
  data.frame(
    "Experiment" = c(
      "PRJNA512997",
      "PRJNA512997",
      "PRJNA512997",
      "PRJNA512997",
      "PRJNA512997",
      "PRJNA512997",
      "PRJNA512997"
    ),
    "SRA" = c(
      "SRR8393424",
      "SRR8393425",
      "SRR8393426",
      "SRR8393427",
      "SRR8393428",
      "SRR8393431",
      "SRR8393432"
    ),
    "Factor" = c(
      "FOXA1",
      "FOXA1",
      "FOXA1",
      "H3K27ac",
      "H3K27ac",
      "None (input)",
      "None (input)"
    ),
    "Tissue" = c(
      "MCF-7LP",
      "MCF-7LP",
      "MCF-7LP",
      "MCF-7LP",
      "MCF-7LP",
      "MCF-7LP",
      "MCF-7LP"
    ),
    "Assembly" = c(
      "GRCh37 Hg19",
      "GRCh37 Hg19",
      "GRCh37 Hg19",
      "GRCh37 Hg19",
      "GRCh37 Hg19",
      "GRCh37 Hg19",
      "GRCh37 Hg19"
    )
  ),
  booktabs = TRUE,
  caption = "Publicly available ChIP-seq SRA files aquired from the NCBI SRA database (accession no. PRJNA512997)."
) %>%
  kable_styling(
    full_width = T,
    latex_options = "hold_position",
    position = "center",
    html_font = "venturis"
  ) %>%
  collapse_rows(columns = c(1, 3, 4, 5),
                valign = "top",
                latex_hline = "none")
```

## EaSeq for the Quantification of FOXA1 and H3K27ac at tRNA genes

EaSeq (v1.111)[@lerdrup2016].

## Motif Analysis

## Statistics

------------------------------------------------------------------------

# Results

## Localisation of FOXA1 at tRNA genes in MCF-7 cells

## Co-localisation of FOXA1 with enhancer marks at tRNA genes

------------------------------------------------------------------------

# Discussion

```{=tex}
\begin{flushright}
`r word_count("report.Rmd") - 2` Words
\end{flushright}
```

------------------------------------------------------------------------

# References
